ldms/src/core/ldms_stream.h

```diff
@@ -129,6 +129,7 @@ struct ldms_stream_client_s {
 /* the full stream message */
 struct ldms_stream_full_msg_s {
 	struct ldms_addr src;
+	uint32_t pad0;
```

Comment:
Just out of curiosity, why are we adding pads in `ldms_stream_full_msg_s` and `ldms_stream_full_msg_s`?

Reply:
I think it is to ensure alignment of the payload on a 32b boundary. But I would suggest we also add one or more 64b "reserved" fields as well to allow us to add additional data to the header for unforeseen features in the future that won't break backward compatibility.